CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+**0.5.1**
+
+* This release includes version **0.5.1** of the [zkVerify](https://github.com/HorizenLabs/zkVerify) project.
```

Comment:
We can not merge this one until the release/tag on zkVerify is ready :(I just looked and it is not yet there)

Reply:
I agree, there's the last run of the CI running; I'll wait for its completion and then create tag there and then merge here.